### ISSUE TITLE
aria attributes for expandable links in Nav

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#746]
A couple of aria attributes should be updated in the expandable link, to be consistent with how we're handling icons in Alerts.

Currently, the attributes `tabindex` and `aria-disabled` are included on the `<span>` that includes the toggle icon, as shown below.

```
    <a href="#" class="pf-c-nav__link" id="expandable-example1" aria-expanded="true">
        Link 1 (current and expanded example)
        <span class="pf-c-nav__toggle" tabindex="-1" aria-disabled="true">
          <i class="fas fa-angle-right"></i>
        </span>
    </a>
```

Instead, these attributes should be removed, and `aria-hidden="true"` should be added to the icon:
```
        <span class="pf-c-nav__toggle">
          <i class="fas fa-angle-right" aria-hidden="true"></i>
        </span>
```

**Comment by rachael-phillips** [comment]
Hi @jgiardino! Can we close this issue?

**Comment by mcoker** [comment]
@rachael-phillips yep, it was fixed in https://github.com/patternfly/patternfly-next/pull/876
